home_work_7/index.js

```diff
@@ -0,0 +1,192 @@
+const validateValue = (value) => {
+  if (value === undefined || value === null) {
+    return;
+  }
+  return String(value);
+};
+
+const EMPTY_VALUE = "Empty field";
+
+/*
+1. Cтворити функцію, яка створює та реєструє нову команду у
+автопробізі. Під час виклику функції потрібно передати дані
+про команду - назва команди, ім'я водія, рік народження водія,
+марка автомобіля, спонсор команди та чи водій допущений до
+автопробігу
+*/
+
+const createAndRegisterNewTeam = (
+  teamName,
+  driverName,
+  driverBirthYear,
+  carBrand,
+  sponsor,
+  driverHasAccess
+) => {
+  const NEW_TEAM = {
+    teamName: validateValue(teamName) || EMPTY_VALUE,
+    driverName: validateValue(driverName) || EMPTY_VALUE,
+    driverBirthYear: validateValue(driverBirthYear) || EMPTY_VALUE,
+    carBrand: validateValue(carBrand) || EMPTY_VALUE,
+    sponsor: validateValue(sponsor) || EMPTY_VALUE,
+    driverHasAccess: Boolean(driverHasAccess),
+  };
+
+  return NEW_TEAM;
+};
+
+createAndRegisterNewTeam();
+// console.log(createAndRegisterNewTeam("Spirit", "John", 1990, "BMW", "HDRezka"));
+
+/*
+2. Запитати дані (через prompt та циклічно) про нового користувача у системі та зберегти його в об'єкт.
+Дані про користувача - логін, пароль, місто, країна, стать, вік
+*/
+
+const createNewUser = () => {
+  const USER = {
+    login: "",
+    password: "",
+    city: "",
+    country: "",
+    sex: "",
+    age: "",
+  };
+
+  for (const [key] of Object.entries(USER)) {
```

Comment:
Also can be in that way
```suggestion
  for (let key of USER) {
```

Reply:
`for (let key of USER) {`  this code doesn't work for me
I got follow error
**Uncaught TypeError: USER is not iterable at**